### PR TITLE
Use external library for FontAwesome widgets

### PR DESCRIPTION
**Pull request by YusukeIwaki**
@RocketChat/android

Just a refactoring.
I created a library for providing FontAwesomeTextView and FontAwesomeButon with `@string/fa_xxx` definitions. (because I use them in 3 hobby projects... 😃  )
Shall we use the library also in Rocket.Chat.Android?

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -82,13 +82,6 @@ play {
     track = "${track}"
 }
 
-repositories {
-    mavenCentral()
-    maven { url 'https://github.com/lijingle1/stetho-realm/raw/master/maven-repo' }
-    maven { url 'http://dl.bintray.com/amulyakhare/maven' }
-    maven { url "https://clojars.org/repo/" } //for icepick.
```
Comment (YusukeIwaki) [inline app/build.gradle:89]: Moving these definition is not directly related to introducing widget-fontawesome. 
However, I think it would be better to define these in one place (root build.gradle).

**Review by laggedHero — APPROVED** [review]
Looks good
